src/fastmcp/utilities/types.py

```diff
@@ -41,6 +41,71 @@ class FastMCPBaseModel(BaseModel):
     model_config = ConfigDict(extra="forbid")
 
 
+@lru_cache(maxsize=5000)
```

Comment:
Oops, this is a duplicate from before I moved this back to dependencies.py, removing...